packages/cyberstorm/src/components/MetaItem.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { ReactNode } from "react";
+import styles from "./componentStyles/MetaItem.module.css";
+
+export interface MetaItemProps {
+  label?: string;
+  icon?: ReactNode;
+  metaItemStyle?: string;
+}
+
+/**
+ * Cyberstorm MetaItem component
+ */
+export const MetaItem: React.FC<MetaItemProps> = (props) => {
+  const { label, icon, metaItemStyle } = props;
+  styles; //if styles is not called, the classes from the css module aren't found
```

Comment:
This is not required. See #569. Rebase this PR on top of that one and use the CSS modules via the imported values rather than strings as `className` values.